### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by NE-SmallTown**

### CODE REVIEW

**Comment by react-linter** [comment]
<details>
  <summary>Details of bundled changes.</summary>

  <p>Comparing: bb627228ea97514acb60901c390f420cdd26d21e...52433dc18d5430b8fb715f20dd72b80c04db5fcc</p>


  
## schedule
File | Filesize Diff | Gzip Diff | Prev Size | Current Size | Prev Gzip | Current Gzip | ENV
 ---  |  ---  |  ---  |  ---  |  ---  |  ---  |  ---  |  --- 
schedule.development.js | n/a | n/a | 0 B | 19.17 KB | 0 B | 5.74 KB | UMD_DEV
**schedule.production.min.js** | **n/a** | **n/a** | **0 B** | **3.16 KB** | **0 B** | **1.53 KB** | **UMD_PROD**

  </details>
  
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: packages/react-dom/src/__tests__/ReactDOMFiberAsync-test.internal.js [hunk 1]
```diff
@@ -341,7 +341,7 @@ describe('ReactDOMFiberAsync', () => {
       ReactDOM.render(<Component />, container);
       // Update
       expect(() => ReactDOM.render(<Component />, container)).toThrow(
-        'flushSync was called from inside a lifecycle method',
+        'flushSync was called from inside a lifecycle method. It cannot be called when React is already rendering.',
```
Comment (gaearon) [inline packages/react-dom/src/__tests__/ReactDOMFiberAsync-test.internal.js]: In general it's fine if we don't test the full message

Comment (NE-SmallTown) [reply packages/react-dom/src/__tests__/ReactDOMFiberAsync-test.internal.js]: Ah, fine, just want keep the consistent

Comment (NE-SmallTown) [reply packages/react-dom/src/__tests__/ReactDOMFiberAsync-test.internal.js]: Wii remove this this evening

Comment (NE-SmallTown) [reply packages/react-dom/src/__tests__/ReactDOMFiberAsync-test.internal.js]: @gaearon done~

Comment (NE-SmallTown) [reply packages/react-dom/src/__tests__/ReactDOMFiberAsync-test.internal.js]: @gaearon done~

**Comment by NE-SmallTown** [comment]
@gaearon done~, just left the fix typo commit

**Comment by NE-SmallTown** [comment]
Should I close this?

**Comment by NE-SmallTown** [comment]
Should I close this?

**Comment by NE-SmallTown** [comment]
@gaearon Any update on this or I miss something?

**Pull request merged**